src/components/BooksList.js

```diff
@@ -0,0 +1,43 @@
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+
+const BooksList = props => {
+  const { books } = props;
+  return (
+    <table>
+      <tbody>
+        <tr>
+          <th>Id</th>
+          <th>Title</th>
+          <th>Category</th>
+        </tr>
+        {books.map(book => <Book key={book.id} book={book} />)}
+      </tbody>
+    </table>
+  );
+};
+
+const Book = ({ book }) => {
+  const { id, title, category } = book;
```

Comment:
- [ ] **[OPTIONAL]** We can directly desctruture objects inside function parameters for brevity and to avoid additional variable declarations.

```js
const Book = ({ id, title, category }) => {
```